GettingStarted.md

```diff
@@ -0,0 +1,54 @@
+# VRChat API Documentation
```

Comment:
Not entirely sure if this one was broken, but just avoid deleting pages, the archive exists for a reason.